controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -220,22 +220,23 @@ module.exports = async (req, res) => {
                             a.is_anonymous = false
                             AND a.is_banned = false
                             AND a.blocked_by_admin = false
-                            AND a.karma_score > 30
+                            AND a.karma_score > 45
                             AND b.topic_id NOT IN (:topics)
                             AND a.user_id NOT IN (:allUserIds)
                           GROUP BY a.user_id
                           ORDER BY 
                             a.last_active_at DESC,
                             CASE WHEN a.profile_pic_path != '%default-profile-picture%' THEN 0 ELSE 1 END,
                             a.followers_count DESC
-                          LIMIT :limit`;
+                          LIMIT :limit OFFSET :offset`;
 
       const userOtherTopicFollowerQueryResult = await sequelize.query(otherTopicsQuery, {
         type: sequelize.QueryTypes.SELECT,
         replacements: {
           topics,
           allUserIds,
-          limit
+          limit,
+          offset
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Ensure that the `offset` value is being correctly passed into the query parameters. It should be validated and sanitized before use.
```diff
- replacements: {
-   topics,
-   allUserIds,
-   limit
- }
+ replacements: {
+   topics,
+   allUserIds,
+   limit,
+   offset
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->